network/wsNetwork.go

```diff
@@ -1488,14 +1544,44 @@ func (wn *WebsocketNetwork) tryConnect(addr, gossipAddr string) {
 	requestHeader.Set(InstanceNameHeader, myInstanceName)
 	conn, response, err := websocketDialer.DialContext(wn.ctx, gossipAddr, requestHeader)
 	if err != nil {
-		wn.log.Warnf("ws connect(%s) fail: %s", gossipAddr, err)
+		if err == websocket.ErrBadHandshake {
+			// reading here from ioutil is safe only because it came from DialContext above, which alredy finsihed reading all the data from the network
+			// and placed it all in a ioutil.NopCloser reader.
+			bodyBytes, _ := ioutil.ReadAll(response.Body)
```

Comment:
Do we really want to readall here?  We should limit its length.

Reply:
I had the same thought; but it doesn't matter. We don't read anything from the network here. It's already in a memory buffer at that point.

Reply:
it matters in as much as it's also being logged to the log file and sent to telemetry.

Reply:
I see; so you don't truly care about the reading. You're concern is that we could end up logging large quantities of information. I'll limit that to 128 characters.